crates/bevy_core_pipeline/src/tonemapping/tonemapping.wgsl

```diff
@@ -1,16 +1,26 @@
 #import bevy_core_pipeline::fullscreen_vertex_shader
-#import bevy_core_pipeline::tonemapping
+
+#import bevy_render::view
 
 @group(0) @binding(0)
-var hdr_texture: texture_2d<f32>;
+var<uniform> view: View;
+
 @group(0) @binding(1)
+var hdr_texture: texture_2d<f32>;
+@group(0) @binding(2)
 var hdr_sampler: sampler;
+@group(0) @binding(3)
+var dt_lut_texture: texture_3d<f32>;
+@group(0) @binding(4)
+var dt_lut_sampler: sampler;
+
+#import bevy_core_pipeline::tonemapping
 
 @fragment
 fn fragment(in: FullscreenVertexOutput) -> @location(0) vec4<f32> {
     let hdr_color = textureSample(hdr_texture, hdr_sampler, in.uv);
 
-    var output_rgb = reinhard_luminance(hdr_color.rgb);
+    var output_rgb = tone_mapping(hdr_color).rgb;
 
 #ifdef DEBAND_DITHER
     output_rgb = pow(output_rgb.rgb, vec3<f32>(1.0 / 2.2));
```

Comment:
Could we use the new `rgb_to_srgb_simple` here? Feel free to close this if it's out of scope. :)

Reply:
Makes sense.

Reply:
I think I'm just going to remove rgb_to_srgb_simple